File: Templates/Challenge/Hello_World/Compute/template.rb
#!/usr/bin/ruby

#
# See ../README.md
#
 
#
# Run as: ruby ch-1.rb [plain | compute]
#

PLAIN   =  0
COMPUTE =  1

type = PLAIN

if   ARGV . length > 0 && ARGV[0] == "compute"
then type = COMPUTE
end

if   type == PLAIN
then puts ("%TEXT");
end

if   type == COMPUTE
then
end

File: Templates/Challenge/Hello_World/Compute/template.js
#!/usr/local/bin/node

//
// See ../README.md
//

//
// Run as: node ch-1.js [plain | compute]
//

let PLAIN   =  0
let COMPUTE =  1

let type = PLAIN

if (process . argv . length > 2 &&
    process . argv [2] == "compute") {
    type = COMPUTE
}

if (type == PLAIN) {
    console . log ("%TEXT")
}

if (type == COMPUTE) {
}
